rise-financial-es6.js

```diff
@@ -27,19 +27,50 @@
           value: () => {
             return [];
           }
+        },
+
+        /**
+         * The id of the display running this instance of the component.
+         */
+        displayId: {
+          type: String,
+          readOnly: true,
+          value: "preview"
         }
+
       };
+
+      this._displayIdReceived = false;
+      this._goPending = false;
     }
 
     _isValidUsage( usage ) {
       return usage === "standalone" || usage === "widget";
     }
 
+    _onDisplayIdReceived( displayId ) {
+      this._displayIdReceived = true;
+
+      if ( displayId && typeof displayId === "string" && displayId !== "" ) {
```

Comment:
@donnapep Agreed I can remove the `!= ""` check. We do have to check the type though because when a polymer component fires an event with no params, the `detail` property of the event is an empty object `{}` which in this case gets passed from https://goo.gl/mscpmx .